### PR TITLE
Implement Basic Mafia Day

### PR DESCRIPTION
**Pull request by shandyk18**
Screenshot 2020-08-10 at 1 46 13 PM - Display 2

### CODE REVIEW

File: ubg/src/mafia/MafiaDay.js [hunk 1]
```diff
@@ -1,12 +1,180 @@
...
+   * Sets up daytime data and logistics
+   * @return {undefined}
+   */
+  function startDay() {
+    if (mafiaKill && mafiaKill.uid !== doctorSave.uid) {
+      usersCollection.doc(mafiaKill).update({alive: false});
+      setDeathText(mafiaKill.displayName + ' was killed by the mafia');
+    } else if (mafiaKill && mafiaKill.uid === doctorSave.uid) {
+      setDeathText(mafiaKill.displayName + ' was saved by the doctor');
+    }
```
Comment (zhangshyue) [inline ubg/src/mafia/MafiaDay.js:57]: When I played, we only show "### is killed tonight" or "no one is killed tonight". I felt like if we provide explicit information would give too much information to the good side. But I'm not sure.

File: ubg/src/pages/Room.js [hunk 2]
```diff
@@ -224,9 +225,10 @@ export default function Room() {
                     user={user}
                     usersData={usersData}
                     room={room}
-                    mafiaKill={roomData.mafiaKill}
-                    doctorSave={roomData.doctorSave}
-                    detectiveCheck={roomData.detectiveCheck}
+                    mafiaKill={usersCollection['mafiaKill']}
+                    doctorSave={usersCollection['doctorSave']}
+                    detectiveCheck={usersCollection['detectiveCheck']}
+                    usersCollection={usersCollection}
```
Comment (zhangshyue) [inline ubg/src/pages/Room.js:231]: Maybe use .mafiaKill instead of ['mafiaKill']

File: ubg/src/mafia/MafiaDay.js (hunk 1 above)
Comment (shandyk18) [reply ubg/src/mafia/MafiaDay.js:57]: Okay I'll make that change!

File: ubg/src/pages/Room.js [hunk 3]
```diff
@@ -224,9 +225,10 @@ export default function Room() {
                     user={user}
                     usersData={usersData}
                     room={room}
-                    mafiaKill={roomData.mafiaKill}
-                    doctorSave={roomData.doctorSave}
-                    detectiveCheck={roomData.detectiveCheck}
+                    mafiaKill={usersCollection.mafiaKill}
```
Comment (danieltre23) [inline ubg/src/pages/Room.js:228]: Isn't usersCollection the ref to to collection?

Comment (shandyk18) [reply ubg/src/pages/Room.js:228]: I wasn't sure about syntax so I followed the comment above. I can change it back.

Comment (zhangshyue) [reply ubg/src/pages/Room.js]: I think it is in roomData though

**Pull request merged**